Documentation/ceph-csi-drivers.md

```diff
@@ -4,261 +4,30 @@ weight: 3200
-	caps: [osd] allow *
-osd.1
-	key: AQBXpa1cTjuYNRAAkohlInoYAa6A3odTRDhnAg==
-	caps: [mgr] allow profile osd
-	caps: [mon] allow profile osd
-	caps: [osd] allow *
-osd.2
-	key: AQB4pa1cvJidLRAALZyAtuOwArO8JZfy7Y5pFg==
-	caps: [mgr] allow profile osd
-	caps: [mon] allow profile osd
-	caps: [osd] allow *
-osd.3
-	key: AQCcpa1cFFQRHRAALBYhqO3m0FRA9pxTOFT2eQ==
-	caps: [mgr] allow profile osd
-	caps: [mon] allow profile osd
-	caps: [osd] allow *
-client.admin
-	key: AQD0pK1cqcBDCBAAdXNXfgAambPz5qWpsq0Mmw==
-	auid: 0
-	caps: [mds] allow *
-	caps: [mgr] allow *
-	caps: [mon] allow *
-	caps: [osd] allow *
-client.bootstrap-mds
-	key: AQD6pK1crJyZCxAA1UTGwtyFv3YYFcBmhWHyoQ==
-	caps: [mon] allow profile bootstrap-mds
-client.bootstrap-mgr
-	key: AQD6pK1c2KaZCxAATWi/I3i0/XEesSipy/HeIA==
-	caps: [mon] allow profile bootstrap-mgr
-client.bootstrap-osd
-	key: AQD6pK1cwa+ZCxAA7XKXRyLQpaHZ+lRXeUk8xQ==
-	caps: [mon] allow profile bootstrap-osd
-client.bootstrap-rbd
-	key: AQD6pK1cULmZCxAA4++Ch/iRKa52297/rbHP+w==
-	caps: [mon] allow profile bootstrap-rbd
-client.bootstrap-rgw
-	key: AQD6pK1cbMKZCxAAGKj5HaMoEl41LHqEafcfPA==
-	caps: [mon] allow profile bootstrap-rgw
-mgr.a
-	key: AQAZpa1chl+DAhAAYyolLBrkht+0sH0HljkFIg==
-	caps: [mds] allow *
-	caps: [mon] allow *
-	caps: [osd] allow *
-
-#get  user key
-sh-4.2# ceph auth get-key client.admin
-AQD0pK1cqcBDCBAAdXNXfgAambPz5qWpsq0Mmw==
-```
-
-Example:
-
-```console
-apiVersion: v1
-kind: Secret
-metadata:
-  name: csi-rbd-secret
-  namespace: default
-stringData:
-  userID: admin
-  userKey: AQD0pK1cqcBDCBAAdXNXfgAambPz5qWpsq0Mmw==
-```
-
-```console
-kubectl create -f cluster/examples/kubernetes/ceph/csi/example/rbd/secret.yaml
-```
-
-## Create RBD PersistentVolumeClaim
-
-Make sure your `storageClassName` is the name of the StorageClass previously
-defined in storageclass.yaml
-
-```console
-kubectl create -f cluster/examples/kubernetes/ceph/csi/example/rbd/pvc.yaml
-```
-
-## Verify RBD PVC has successfully been created
-
-```yaml
-# kubectl get pvc
-NAME      STATUS   VOLUME                 CAPACITY   ACCESS MODES   STORAGECLASS   AGE
-rbd-pvc   Bound    pvc-c20495c0d5de11e8   1Gi        RWO            csi-rbd        21s
-```
-
-If your PVC status isn't `Bound`, check the csi-rbdplugin logs to see what's
-preventing the PVC from being up and bound.
-
-## Create RBD demo Pod
-
-```console
-kubectl create -f cluster/examples/kubernetes/ceph/csi/example/rbd/pod.yaml
-```
-
-When running `rbd list block --pool [yourpool]` in one of your Ceph pods you
-should see the created PVC:
-
-```bash
-# rbd list block --pool rbd
-pvc-c20495c0d5de11e8
-```
-
-## Additional features
-
-### RBD Snapshots
+## RBD Snapshots
 
 Since this feature is still in [alpha
 stage](https://kubernetes.io/blog/2018/10/09/introducing-volume-snapshot-alpha-for-kubernetes/)
 (k8s 1.12+), make sure to enable `VolumeSnapshotDataSource` feature gate in
 your Kubernetes cluster.
 
-#### create RBD snapshot-class
+### SnapshotClass
```

Comment:
Do we need to mention about `SnapshotClass` as `snapshot` support is in alpha state. @Madhu-1 what do you think?